apps/studio/components/interfaces/Auth/Policies/Policies.tsx

```diff
@@ -145,7 +145,7 @@ const Policies = ({
             <section key={table.id}>
               <PolicyTableRow
                 table={table}
-                isLocked={isLocked}
+                isLocked={schema === 'realtime' ? true : isLocked}
```

Comment:
shall we just add realtime schema here?
https://github.com/supabase/supabase/blob/master/apps/studio/lib/constants/schemas.ts

Reply:
it's already there but I need to do some manual overrides to not show the alert (because it's not true anymore)

![Screenshot 2024-06-25 at 15 16 32](https://github.com/supabase/supabase/assets/568291/5fd96fa6-b846-4a40-bc2f-48c0b08c47da)

but at the same time, make the remaining two tables in `realtime` schema locked.

Reply:
ahhh okay i see what you mean - only the messages table is "unlocked"